src/main/java/yevhenii/lostfilmdemo/kafka/TVSeriesReader.java

```diff
@@ -0,0 +1,45 @@
+package yevhenii.lostfilmdemo.kafka;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+import yevhenii.lostfilmdemo.controllers.OmdbHolderClient;
+import yevhenii.lostfilmdemo.convertors.TVRecordConvertor;
+import yevhenii.lostfilmdemo.entity.ImdbEpisodesDTO;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+import yevhenii.lostfilmdemo.repository.TVSeriesRepository;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class TVSeriesReader {
+    @Value("${omdb.api.key}")
+    private final String apiKey;
+    private final OmdbHolderClient omdbHolderClient;
+    private final TVSeriesRepository repository;
+    private final TVRecordConvertor convertor;
+
+    @KafkaListener(topics = {"${lostfilm.topic}"})
+    public void receive(ConsumerRecord<String, TVSeries> consumerRecord) {
```

Comment:
you can skip ConsumerRecord if you don't need key value, just change it to `receive(TVSeries record)`